models/tokens/bnb/tokens_bnb_schema.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+models:
+  - name: tokens_bnb_bep20
+    meta:
+      blockchain: bnb
+      sector: tokens
+      project: bep20
+      contibutors: hildobby
+    config:
+      tags: ['table', 'bep20', 'bnb']
```

Comment:
please add 'metadata' tag here, it'll help us do deployments quicker on hardcoded metadata views
reference to open PR on existing: https://github.com/duneanalytics/spellbook/pull/1438/files

Reply:
also, 'table' should be 'tokens' i believe

Reply:
Done! ✅